### DISCUSSION TITLE
Libre app not necessary anymore - writing help needed

### DISCUSSION THREAD

**Discussion by twinko** [maheini/FreeStyle-Libre-3-patch#50] [General]
Hey by accident i scanned a new sensor directly with juggluco. Everything works like before, no need to switch back and forth between libre3 app and juggluco.

![Screenshot_20230626_213326_Juggluco](https://github.com/maheini/FreeStyle-Libre-3-patch/assets/11973158/a26ef582-a9c8-4853-ae75-0a7da1250a99)

This will require a change in the description. Would be great if someone else has the time. If not. Ill send changes asap

**Comment by maheini** [comment]
Hi @twinko 
You're absolutely right - it's already possible to start a sensor with Juggluco since a long time - however, I got no time. Totally in a hurry right now, since my business skyrocket this spring. Maybe I will get some time in the autumn, but well see.

Thank you anyways for those news - maybe they will help some people making the process easier.
Have a nice day - maheini

**Reply by twinko** [reply]
Hey @maheini that's awesome to hear, whish you all the best for your business!
if i find the time ill send either text snippets you could copy past or send pull request :)

**Reply by maheini** [reply]
👍

**Reply by twinko** [reply]
send a pull request. didnt had the time to reread it carefully. but should be good enough to stop people from using the patched libre app ^^
